handbook/engineering/README.md

```diff
@@ -726,7 +732,18 @@ If using the standard bug report, the bug is labeled “bug” and “reproduce.
 #### Reproduced
 QA has reproduced the issue successfully. It should now be transferred to engineering. 
 
-Remove the “reproduce” label, add the label of the relevant team (e.g. #cx, #mdm, #compliance), add the "product" label, and assign it to the relevant engineering manager. (Make your best guess as to which team. The EM will re-assign if they think it belongs to another team.) [See on GitHub](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+label%3Abug+label%3A%3Aproduct%2C%3Arelease+-label%3A%3Areproduce+sort%3Aupdated-asc+).
+Remove the “reproduce” label, add the label of the relevant team (e.g. #g-cx, #g-mdm, #g-infra, #g-website), and assign it to the relevant engineering manager. (Make your best guess as to which team. The EM will re-assign if they think it belongs to another team.) [See on GitHub](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+label%3Abug+label%3A%3Aproduct%2C%3Arelease+-label%3A%3Areproduce+sort%3Aupdated-asc+).
+
+#### In product drafting (as needed)
+If a bug requires input from product, the `:product` label is added and the bug is moved to the "Product drafting" column of the (bugs board)[https://app.zenhub.com/workspaces/-bugs-647f6d382e171b003416f51a/board]. It will stay in this state until product closes the bug, or removes the `:product` label, moves the back to the "Inbox" on the bugs board, and assigns it to the appropriate EM.
+
+#### In engineering
+A bug is in engineering after it has been reproduced and assigned to an EM. The EM will determine if it meets the criteria for a [critical bug](https://fleetdm.com/handbook/engineering#critical-bugs) or a [release blocking bug](https://fleetdm.com/handbook/engineering#release-blocking-bugs). If so, the `:release` label is added, along with `~relaser blocker` or `~critical bug`, and it is moved to the "Current release' column of the bugs board. If the bug does not meet the criteria, the EM will determine if there is capacity in the current sprint for this bug. If so, the `:release` label is added, and it is moved to the "Current release' column on the bugs board. If there is no available capacity in the current sprint, the EM will move the bug to the "Sprint backlog" column where it will be prioritized for the next sprint. 
```

Comment:
The EM deciding whether it is a critical bug is different from what is described in the critical bug notification process, which says product decides. Open to either, but let's get clarity.